actions/websocket_actions.jsx

```diff
@@ -182,6 +182,13 @@ export function unregisterPluginReconnectHandler(pluginId) {
     Reflect.deleteProperty(pluginReconnectHandlers, pluginId);
 }
 
+function restart() {
+    reconnect(false);
+
+    // We fetch the client config again on the server restart.
+    getClientConfig()(dispatch, getState);
```

Comment:
Could you switch this around? These generally behave the same, but dispatching the result of `getClientConfig` works regardless of whether or not `getClientConfig` returns a thunk action or a plain redux action.
```suggestion
    dispatch(getClientConfig());
```

Reply:
yes thats makes sense